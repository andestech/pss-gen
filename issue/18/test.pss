/*!
@weakgroup      issue
@{
@file
@author         Luther Lee
@data           2024/05/08
@brief          Test issue 18.


---------------------------------------------------------------------------------------------------

### Test Command
@code{.unparsed}
make
@endcode

@cond
*/
component pss_top {
    action root_a {
        activity {
            do printStr with { str == "one ;"; };
            do printStr with { str == "two ;"; };;
            do printStr with { str == "three ;"; };;;
        }
    }

    action printStr {
        rand string str;
        exec body ASM = """printStr: {{str}}""";
    }
}

/**@}*/
