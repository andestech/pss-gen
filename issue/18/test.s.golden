printStr: one ;
printStr: two ;
printStr: three ;
